hack/migrate-to-antrea.sh

```diff
@@ -0,0 +1,103 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used for migrating other CNIs to Antrea CNI.
+
+set -e
+
+_usage="Usage: $0 [--dry-run] [--kubeconfig] [--antrea-manifest] [-h | --help]
+Migrate other CNIs to Antrea. During the migration, the workloads will be restarted and the network will be interrupted.
+        --dry-run                   Print the commands that would be executed, but do not execute them.
+        --kubeconfig <KUBECONFIG>   Path to kubeconfig file.
+        --antrea-manifest <PATH>    Path to the Antrea manifest file, if not specified, the latest manifest will be used.
+"
+
+_warning_message="Warning: Services might be unavailable during migration
+Please make sure that you want to continue
+Input 'yes' to continue"
+
+_completed_message="The migration is completed, Antrea takes over the network management now.
+Please make sure that you have removed the old CNI plugin and the old CNI configuration file."
+
+DRY_RUN=false
+KUBECONFIG="$HOME/.kube/config"
+ANTREA_MANIFEST="https://raw.githubusercontent.com/antrea-io/antrea/master/build/yamls/antrea.yml"
+
+function print_usage {
+    echo "$_usage"
+}
+
+while [[ $# -gt 0 ]]; do
+    key="$1"
+
+    case $key in
+    --dry-run)
+        DRY_RUN=true
+        shift
+        ;;
+    --kubeconfig)
+        KUBECONFIG="$2"
+        shift 2
+        ;;
+    --antrea-manifest)
+        ANTREA_MANIFEST="$2"
+        shift 2
+        ;;
+    -h | --help)
+        print_usage
+        exit 0
+        ;;
+    *) # unknown option
+        echo "Unknown option $1"
+        exit 1
+        ;;
+    esac
+done
+
+echo "$_warning_message"
+read -r input
+if [ "$input" != "yes" ]; then
+    echo "========== Aborting =========="
+    exit 1
+fi
+
+echo "========== Install Antrea =========="
+if [ $DRY_RUN == true ]; then
+    echo "kubectl apply -f $ANTREA_MANIFEST --kubeconfig $KUBECONFIG"
+    echo "kubectl rollout status ds/antrea-agent -n kube-system --timeout=300s --kubeconfig $KUBECONFIG"
+else
+    trap "kubectl delete -f $ANTREA_MANIFEST --kubeconfig $KUBECONFIG" ERR
+    kubectl apply -f $ANTREA_MANIFEST --kubeconfig "$KUBECONFIG"
+    kubectl rollout status ds/antrea-agent -n kube-system --timeout=300s --kubeconfig "$KUBECONFIG"
+fi
+
+
+echo "========== Restarting the sandboxes =========="
+if [ $DRY_RUN == true ]; then
+    echo "kubectl apply -f ./build/yamls/antrea-migrator.yml --kubeconfig $KUBECONFIG"
+else
+    trap "kubectl delete -f ./build/yamls/antrea-migrator.yml --kubeconfig $KUBECONFIG" EXIT
+    kubectl apply -f ./build/yamls/antrea-migrator.yml --kubeconfig "$KUBECONFIG"
+fi
+if [ $DRY_RUN == true ]; then
+    echo "kubectl rollout status ds/antrea-migrator -n kube-system --timeout=300s --kubeconfig $KUBECONFIG"
+else
+    echo "Waiting for sandboxes restart to be finished"
+    kubectl rollout status ds/antrea-migrator -n kube-system --timeout=300s --kubeconfig "$KUBECONFIG"
```

Comment:
Should this migrator DaemonSet being kept in the cluster?  or it should exit after migration is done?